web/src/index.html

```diff
@@ -137,14 +137,11 @@ <h2 class="modal-title">Restore Settings</h2>
     </div>
   </div>
 
+  <nav class="navbar navbar-expand-lg">
+    <a class="navbar-brand" href="https://github.com/sidoh/esp8266_milight_hub">MiLight Hub</a>
```

Comment:
I am used to the brand link pointing at `/` (e.g., being equivalent to a refresh).

Linking to the Github page seems like a good idea.  Maybe a pulled right Github logo would make more sense?